src/OpenTelemetry.Instrumentation.Cassandra/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
-## Unreleased
+## 1.0.0
```

Comment:
Consider to release as 1.0.0-alpha.1/1.0.0-beta.1 or even (0.1.0-beta.1). to signalize that it is brand new package and it was not tested in the wild.

I think that we should not release this as a stable before https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/database-metrics.md reach stability (and this package will implement this metrics).

Reply:
Done